### DISCUSSION TITLE
Welcome to librelink Discussions!

### DISCUSSION THREAD

**Discussion by gillesvs** [gillesvs/librelink#9] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by SeeWhatHappens** [comment]
I had this working for a week or two, has since stopped working. I have tried a few things to resolve, it would be helpful to get confirmation from another user that librelink still works.
